### PR TITLE
Trade offs should be clear in the specifications

### PR DESCRIPTION
**Pull request by AramZS**
Objections included concerns that the charter did not list trade offs in utility vs privacy that would be considered during operation and consideration of specifications. Since specifications may have different concerns around utility and privacy it seems inappropriate to try and list these in the charter. Instead this change requires those concerns to be addressed on a per proposal and feature basis.

See https://github.com/patcg/patwg-charter/issues/44 to help with understanding this issue.

### CODE REVIEW

File: charter.html [hunk 1]
```diff
@@ -268,10 +268,14 @@ <h2>Success Criteria</h2>
     specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
 
           <p>Each normative specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+          
+          <p>Any proposed feature should reduce the data available about a user.</p>
```
Comment (jaylett-annalect) [inline charter.html:272]: Does this belong in this PR? It seems unconnected. Additionally: presumably this isn't _strictly_ reduce (ie features that don't change the amount of data available about a user are fine)?

Comment (brodrigu) [reply charter.html:272]:
```suggestion
          <p>Any proposed feature should increase the protection of data about a user.</p>
```

Comment (brodrigu) [reply charter.html:272]:
```suggestion
          <p>Any proposed feature should increase the protection of data about a user.</p>
```

**Comment by AramZS** [comment]
I am relaying a comment from a member who has raised formal objections. I do not intend to imply support or opposition to this statement, but just to make their objection on this available for discussion with the member's permission: 

> The proposed changes take the charter in a better direction. However, protection must be aligned to the agreement that the user entered into with the data controller and be proportional to the risk-based approach advocated by GDPR. Taking a direction where the web browser could learn the nature of that agreement and take steps to alert the user to non-compliance would be a very positive step.

File: charter.html (hunk 1 above)
Comment (AramZS) [reply charter.html:272]: @jaylett-annalect The goal here is to provide an outline for exactly what the risk involved is (users' data being emitted) for the question of utility and safeguards below.  

> features that don't change the amount of data available about a user are fine

The argument I would make is that while those features that maintain the status quo of data flow between the user and site might or might not be fine (depending on the feature), those features that do not explicitly reduce the flow of user data, or give a user the ability to choose to reduce the flow of data don't really belong in this group's workflow, since they wouldn't be accomplishing the Privacy part of Private Ad Technology. This specific separation is what I'm hoping will be clear here. We don't want to create an overly broad scope for the group.

Comment (jaylett-annalect) [reply charter.html:272]: I may be being overly pedantic, but something like IPA is not changing the volume of user data flowing around. It's introducing data that would not be considered user data (assuming the privacy guarantees hold up!). But it by itself does nothing to _reduce_ flows; it relies on (for instance) third-party cookie deprecation to achieve that (which is already in place on some browsers, so not reducing the flow of data there either). Or I suppose it could rely on deprecation of current attribution techniques, inclusive of those dependent on 3PC. So perhaps it could be "any proposed feature should reduce the data available about a user compared to current techniques and practice" ... although that's a bit fluffy perhaps.

I understand and support the goal here, I'm just not sure the text is right.

File: charter.html [hunk 2]
```diff
@@ -268,10 +268,14 @@ <h2>Success Criteria</h2>
...
 
           <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
 
     <p>Normative specifications which have user-facing features should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
+        
+          <p>In considering features, the group will state, during the development of those features or in the text of the document, what utility is aimed to be provided and what specific risks it aims to address. Proposals should clearly state business factors and user risks they will consider and how they will decrease such risks and what safeguards will be put in place to decrease such risk and continue to enable the business factors they wish to preserve</p>
```
Comment (eriktaubeneck) [inline charter.html:278]:
```suggestion
          <p>In considering features, the group will state, during the development of those features or in the text of the document, what utility is aimed to be provided and what specific risks it aims to address. Proposals should clearly state business factors and user risks they will consider and how they will decrease such risks and what safeguards will be put in place to decrease such risk and continue to enable the business factors they wish to preserve.</p>
```

**Comment by AramZS** [comment]
Separate two line changes into two separate PRs

**Comment by AramZS** [comment]
General agreement seems to move towards removing 272 in general.

File: charter.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (michaelkleber) [reply charter.html:272]: I agree with Martin's objection, and moreover I do not believe there is any way to rephrase this to handle that problem.  Strike.

File: charter.html [hunk 3]
```diff
@@ -268,10 +268,14 @@ <h2>Success Criteria</h2>
...
 
           <p>Each normative specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+          
+          <p>Any proposed feature should reduce the data available about a user.</p>
 
           <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
 
     <p>Normative specifications which have user-facing features should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
+        
+          <p>In considering features, the group will state, during the development of those features or in the text of the document, what utility is aimed to be provided and what specific risks it aims to address. Proposals should clearly state business factors and user risks they will consider and how they will decrease such risks and what safeguards will be put in place to decrease such risk and continue to enable the business factors they wish to preserve.</p>
```
Comment (eriktaubeneck) [inline charter.html:278]:
```suggestion
          <p>In considering features, the group will state, during the development of those features or in the text of the document, what functionality is aimed to be provided and what specific risks it aims to address. Proposals should clearly state business factors and user risks they will consider and how they will decrease such risks and what safeguards will be put in place to decrease such risk and continue to enable the business factors they wish to preserve.</p>
```

**Comment by AramZS** [comment]
Noting: 

- ln 272 is split out into #58 
- Remainder is split out into #59 

Please continue any discussion on those threads.
